source/_integrations/actron_air.markdown

```diff
@@ -37,6 +38,10 @@ The integration will create a climate entity for the main air conditioning syste
 
 Each zone will be reflected as a separate climate entity. You can set the temperature and operation mode per zone (if supported by your air conditioner).
 
+### Switch
+
+The integration will add the away mode, continuous fan, quiet mode, and turbo mode (if supported) switches to your Actron Air air conditioner.
```

Comment:
```suggestion
The integration will add the **Away** mode, **Continuous fan**, **Quiet** mode, and **Turbo** mode (if supported) switches to your Actron Air air conditioner.
```
Are these UI strings? If so, they can be marked bold. If not, please ignore my comment.